news/2024/2024-09-17-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,175 @@
+
+Put another way, missing in long beatmaps with a single "difficulty spike" will be punished harder than in long maps that are consistently difficult. The algorithm essentially assumes that misses happen on the most difficult parts.
+
+As touched on earlier, big jumps in total pp are expected in this round. The removal of combo scaling will result in large increases for users who have many scores with lower combos that are considered difficult. It may also result in decreases for users who have many scores with higher combos but saturated a lot of misses towards the end of a beatmap.
+
+### Rhythm complexity improvements
+
+Rhythm complexity can be expressed as "changes in pattern lengths and timings" — quintuples changing into triples, even bursts into odd bursts, 1/4 streams into 1/3 streams, etc. More erratic changes imply a more complex rhythm, making it harder for players to adapt and recognise the rhythm they should tap to.
+
+A [change](https://github.com/ppy/osu/pull/28871) proposed by [StanR](https://osu.ppy.sh/users/7217455) fixes some flaws in osu!'s rhythm complexity calculations:
+
+- Highly complex rhythm has received a general buff, affecting maps such as [II-L - Asteroid Field of DECAPLETS](https://osu.ppy.sh/beatmapsets/1666685#osu/3403124).
+- A bonus was previously applied based on pattern length, which caused long bursts to be assessed as rhythmically complex even if they were not. This has been removed.
+- "Double-tappable" patterns have been nerfed, i.e. overlapping notes where the time between objects is low enough to comfortably double-tap and still receive 300s. This double-tap nerf already existed in the speed portion of pp, and this change now brings it over to rhythm complexity calculations too.
+- Repeated note groups have been heavily nerfed. This practically means that patterns such as repeated triple spam (like in [nameless - SLoWMoTIoN](https://osu.ppy.sh/beatmapsets/295848#osu/988793)) are nerfed significantly:
+
+![](/wiki/shared/news/2024-09-17-performance-points-star-rating-updates/island-repetition-nerf.png)
+
+### Speed distance hotfix
+
+A [change](https://github.com/ppy/osu/pull/29980) proposed by [tsunyoku](https://osu.ppy.sh/users/11315329) brings a minor change to a bonus applied for speed distance bonus.
+
+The distance bonus exists to reward flow aim by multiplying it with the speed difficulty. In this hotfix, rather than the bonuses being multiplicative, they have been changed to be additive to decrease the effect the distance bonus has on the speed value.
+
+In addition, the distance bonus scaling has been changed to decrease the reward for lower-spacing flow aim. Finally, a new multiplier was added to the final result of the distance bonus to decrease it overall.
+
+![](/wiki/shared/news/2024-09-17-performance-points-star-rating-updates/distance-bonus.png)
```

Comment:
```suggestion
### Flow aim hotfix

Fast flow aim (more than 200 BPM) is rewarded by taking into account the distance between objects and the general speed difficulty. These values used to be multipliers, but they're now additive instead. In effect, this nerfs fast spaced streams significantly.

The same [change](https://github.com/ppy/osu/pull/29980) by [tsunyoku](https://osu.ppy.sh/users/11315329) additionally performs other minor changes to how these values are calculated, lowering them overall, especially for lower-spaced flow aim.
```

Reply:
Not a fan of that first sentence. It's true that the actual speed bonus is only applied at BPMs >= 200, but the distance bonus is a separate thing that applies regardless of the BPM. The actual speed bonus is also not about flow aim, while the distance bonus is. I think it would be better to remove the BPM mention entirely

Reply:
ty for clarifying, i've edited it out